### DISCUSSION TITLE
🧩 FEATURE REQUEST: Make the Companion Widget draggable

### DISCUSSION THREAD

**Discussion by Posandu** [dailydotdev/daily#622] [Feature Requests]
### Description
Make the companion widget draggable, because the content behind the widget is not visible.

### Solution
Maybe something like this, 
![image](https://user-images.githubusercontent.com/76736580/171909010-06607903-0b90-4916-a689-cd7e00fbb734.png)
By moving the mouse over the dots in the bottom, we can drag the widget.

**Comment by idoshamun** [comment]
Hi @Posandu,

Thank you for this awesome suggestion! I added it to our companion v2 backlog. 😬
